src/app/plugins/cointoss.plugin.ts

```diff
@@ -0,0 +1,36 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+import { MessageEmbed } from 'discord.js';
+
+export class CoinToss extends Plugin {
+  public name: string = 'Coin Toss Plugin';
+  public description: string = 'Ask Lion to toss a coin or if given arguments, choose among the arguments.';
+  public usage: string = '!cointoss || !pick <arg_1> <arg_2> ... <arg_n>';
+  public pluginAlias: string[] = ['pick', 'select', 'cointoss', 'coin', 'toss', 'ct'];
+  public permission: ChannelType = ChannelType.Public;
+  private _embed: MessageEmbed = new MessageEmbed();
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    let _responses: string[] = [];
+    let _image: string = '';
```

Comment:
Make the array a const, you can push to an array that is constant
then you can just push with the spread operator
```
repsonses.push(...args)
```

Reply:
Also we name things with a hanging underscore when they are a private property/method on a class, so no need to do that here